tools/versions.gradle

```diff
@@ -0,0 +1,31 @@
+ext.versions = [
+

```

Comment:
What is the naming pattern here? cammelCase or snake_case? `androidSupport` x `android_plugin`

Reply:
Done! pattern snake case.